src/drivers/uavcan/uavcan_main.cpp

```diff
@@ -429,8 +452,14 @@ UavcanNode::start(uavcan::NodeID node_id, uint32_t bitrate)
 	}
 
 	_instance->ScheduleOnInterval(ScheduleIntervalMs * 1000);
+
+#if defined(CONFIG_UAVCAN_ESC_CONTROLLER)
 	_instance->_mixing_interface_esc.ScheduleNow();
+#endif
+
+#if defined(CONFIG_UAVCAN_SERVO_CONTROLLER)
 	_instance->_mixing_interface_servo.ScheduleNow();
+#endif
```

Comment:
I would keep these together under a single option, as they're exposed together in the UI as well.